.drone.yml

```diff
@@ -20,4 +20,14 @@ steps:
     - pull_request
     - push
 
-...
\ No newline at end of file
+- name: unit_tests
+  pull: if-not-exists
+  image: python:3.9-slim-buster
```

Comment:
Could be useful to bump this to 3.10, if possible. This depends on the Python version we're running in prod, please make sure to check runtime environment.

Reply:
If runtime running lambda is Python 3.9, please keep as it is.